Godeps/_workspace/src/github.com/syncthing/protocol/message.go

```diff
@@ -28,6 +30,9 @@ func (f FileInfo) String() string {
 }
 
 func (f FileInfo) Size() (bytes int64) {
+	if f.ActualSize > 0 {
+		return f.ActualSize
+	}
 	if f.IsDeleted() || f.IsDirectory() {
 		return 128
```

Comment:
Seemed like a nice round number.

(Seriously though, it just represents a small nonzero value, to indicate that there is some work to be done and cause us to be out of sync when we are "missing" deleted files and so on.)